json-logic/README.md

```diff
@@ -0,0 +1,62 @@
+# CovQuestions Draft
+
+This is the draft of a custom questionaire format for the CovQuestions project.
+
+## Requirements
+
+* Relatively easy to edit
+* Easy to use and implement
+* As platform-independend as possible
+* Describe questions, results, and result-computation logic
+* Questions might be shown or not depending in intermediate results
+* Support for multiple result categories, with their own results and computations
+
+## Considerations
+
+### Result Format
+
+Two possible options: 
+1. "Flat" list of results with a expressione evaluating to a boolean
+2. Result categories with an expression that evaluate to a result id with a list of results per category.
+
+We favor option 2 - with option 1 it is easy to accidentially show two results from the same category. This is impossible with 2 by design at the price of one additional layer of nesting.
```

Comment:
Can you elaborate or give an example?

Reply:
Example why flat might be bad:
Imagine you have four possible risk categories, and each has their own expression, yielding true or false.
Now, by accident, there is a bug, and two of these answer expressions evaluate to true: Two contradicting answers show. 

Using categories of answers, with one expression *per category* makes that behavior impossible by design.

Reply:
I like the category thinking, but I can't see it solving the problem. 
While it makes it easier to understand it's still possible to have multiple risk categories returned? 
Just that now each category has to return a true. 
Maybe we should talk about that via phone. I think I did not wrap my mind around the concept yet.

Reply:
Yes, let's talk it through on phone.